src/sentry/web/frontend/organization_auth_settings.py

```diff
@@ -168,7 +168,7 @@ def handle_existing_provider(
                 changed_data = {}
                 for key, value in form.cleaned_data.items():
                     if key == "x509cert":
-                        original_idp = form.initial.get("idp", {})
```

Comment:
Initially i was putting the original idp data on the form

but since we're not using that data on the form - we can just pull it out from the auth_provider here